webapp/channels/src/packages/mattermost-redux/src/selectors/entities/posts.ts

```diff
@@ -800,3 +803,18 @@ export function makeGetPostAcknowledgementsWithProfiles(): (state: GlobalState,
         },
     );
 }
+
+export function getTeamIdFromPost(state: GlobalState, postId: Post['id']): Team['id'] {
+    const post = getPost(state, postId);
+    const channel = getChannel(state, post.channel_id);
+
+    if (!channel) {
+        return '';
+    }
```

Comment:
I don't really like this, since it has double meaning.

Reply:
I'd be totally fine if we revert that then. I was only 0/5 on making it a string anyway